README.md

```diff
@@ -25,35 +25,29 @@ go version
 ### Download the new libwasmvm.x86_64.so:
 
 ```sh
-wget https://github.com/CosmWasm/wasmvm/releases/download/v1.5.2/libwasmvm.x86_64.so
+wget https://github.com/CosmWasm/wasmvm/releases/download/v1.5.2/libwasmvm.x86_64.so -O /usr/lib/libwasmvm.x86_64.so
```

Comment:
The `-o /usr/lib/libwasmvm.x86_64.so` tries to write directly to a system directory (`/usr/lib/`), which usually requires elevated permissions.

The better approach is this:
```sudo mv libwasmvm.x86_64.so /usr/lib/```